app/js/questions.ts

```diff
@@ -25,6 +27,19 @@ function fillUserTable(HBTemplate: string) {
             fillNav()
             active()
             changeQuestion(current)
+                if (canResumeCookie == '1') {
+                    getAnswersToResume(uid).then(result => {
+                        document.querySelectorAll<HTMLElement>('.question').forEach(question => {
+                            question.querySelectorAll<HTMLInputElement>('.questionValue').forEach(questionValue => {
+                                let data = result.answers[question.dataset.id].answerID
+                                if (data !== undefined && questionValue.value == data) {
+                                    question.querySelector('#notes').textContent = result.answers[question.dataset.id].notes
```

Comment:
if you store `result.answers[question.dataset.id]` in a variable instead of with the answerID, you could reuse that variable here as well, making the code tidier